Support_Your_Locals/Views/Business/AddAdvertisement.cshtml

```diff
@@ -70,14 +72,17 @@
                     <div class="form-group">
                         <label asp-for="Header">Header</label>
                         <input asp-for="Header" class="form-control">
+                        <span asp-validation-for="Header"></span>
```

Comment:
You don't need to hardcode this. Just use     <div class="text-danger" asp-validation-summary="All"></div>
This will act as <span asp-validation-for="Header"></span> and <span asp-validation-for="Description"></span> and so on.